src/Monitoring.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
We are always going to need this with long-running processes but it could be extracted on its own. For now, it's fine here as `eLife\Bus\Monitoring`.